### PR TITLE
added myself

### PR DESCRIPTION
**Pull request by yasoob**

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -217,14 +217,14 @@
...
-  
+
 - name: Shreyas Sanghvi
   github: shreyassanghvi
   image: shreyassanghvi.jpg
   blog: http://shreyassanghvi.co.in/
   twitter: shreyas2101
   facebook: shreyasharshant
-  bio: 16, Aviator, coder
+  bio: "16, Aviator, coder"
```
Comment (roonyh) [inline _data/students.yml]: Remove those quotes. We don't them.

File: _data/students.yml [hunk 2]
```diff
@@ -233,4 +233,13 @@
...
+  bio: Coding is passion 
+
+- name: M.Yasoob Ullah Khalid
+  github: yasoob
+  image: yasoob.jpg
+  blog: http://yasoob.me
+  twitter: yasoobkhalid
+  facebook: m.yasoob
+  stackexchange: 
+  bio: "17, Author, Blogger, Programmer. First & last GCI."
```
Comment (roonyh) [inline _data/students.yml]: Remove those quotes

**Comment by roonyh** [comment]
Good job! :+1: We can merge this when you fix the small issues :)

**Comment by yasoob** [comment]
Done :)

**Comment by mananwason** [comment]
@samarjeet27 Please don't merge commits that aren't squashed.

**Comment by mananwason** [comment]
@yasoob Please squash commits and create a new Pull request.

**Comment by yasoob** [comment]
Ok :)

**Pull request merged**